File: Cuaderno_1/ej12.py
"""
El salario base de un vendedor es de 2.000 euros mensuales. A este salario
se le suma un 3% de comisión sobre el total de las ventas que ha realizado,
pero al total obtenido hay que descontarle un 32% del IRPF. Escribe un
programa que, a partir del importe de las ventas que ha realizado el vendedor
durante el último mes y escriba el salario neto que cobrará ese mes.

"""

importeVentas = float(input("Introduce cuánto has vendido este mes: "))

salario = ( 2000 + (importeVentas + importeVentas * 0.03) ) * (1 - 0.32)

print("Este mes vas a cobrar:",salario,"euros")

File: Cuaderno_1_JAVA/ej12.java

/*importeVentas = float(input("Introduce cuánto has vendido este mes: "))

salario = ( 2000 + (importeVentas + importeVentas * 0.03) ) * (1 - 0.32)

print("Este mes vas a cobrar:",salario,"euros")*/
import java.util.Scanner;

public class ej12 {
    public static void main(String[] args) {
        Scanner read = new Scanner(System.in);
        System.out.println("Cuanto has vendido este mes??");
        float importeVentas = Float.parseFloat(read.nextLine());
        read.close();
        float salario = (2000 + (importeVentas + importeVentas * 0.3f)) * (1 - 0.32f);
        System.out.println("Este mes vas a cobrar: "+salario+" EUR");
    }
}
